### DISCUSSION TITLE
No module named 'pycuda.compyte'

### DISCUSSION THREAD

**Discussion by baihaozi12** [inducer/pycuda#339] [Troubleshooting]
**Describe the bug**
When I install pycuda and run 'import pycuda.autoinit'. and the terminal shows that " from pycuda.compyte.dtypes import (  # noqa: F401   ModuleNotFoundError: No module named 'pycuda.compyte' "

**To Reproduce**
1. install pycuda 
2. 'import pycuda.autoinit

**Expected behavior**
'import pycuda.autoinit'
from pycuda.compyte.dtypes import (  # noqa: F401   ModuleNotFoundError: No module named 'pycuda.compyte' 

**Environment (please complete the following information):**
 - OS: Jetson, ubuntu 18.04
 - CUDA version: [e.g. 10.4]
  - PyCUDA version: [e.g 2021.1]
  - Python version: [e.g. 3.6.9]

**Additional context**

**Comment by inducer** [comment]
You downloaded the release from the (very broken) Github "Releases" tab. Download the release from https://pypi.org/project/pycuda/ instead.

**Comment by lyz678** [comment]
子模块未下载
用git下载
git clone --recursive https://github.com/inducer/pycuda.git
或者手动下载
https://github.com/inducer/compyte.git
